### PR TITLE
platformmanagement_public_514 related json files

### PR DESCRIPTION
**Pull request by wjiangjay**

### CODE REVIEW

File: image-streams/tc510523.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+       "name": "python",
+
+       "creationTimestamp": null
+
+    },
+
+    "spec": {
+
+       "dockerImageRepository": "wsun/python-33-centos",
```
Comment (wsun1) [inline image-streams/tc510523.json]: Please don't use personal image

**Comment by wjiangjay** [comment]
@wsun1  replace the individual repo to shared one

**Comment by liangxia** [comment]
Are the empty lines necessary ?

File: image-streams/tc510525.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+
+    "kind": "ImageStream",
+
+    "apiVersion": "v1",
+
+    "metadata": {
+
+      "name": "python",
```
Comment (wsun1) [inline image-streams/tc510525.json]: Your image is ruby,but your name is python,or you could use more meaningful name?

File: image-streams/tc510523.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+
+    "kind": "ImageStream",
+
+    "apiVersion": "v1",
+
+    "metadata": {
+
+       "name": "python",
```
Comment (wsun1) [inline image-streams/tc510523.json]: Your image is ruby,but your name is python,or you could use more meaningful name?

**Comment by wjiangjay** [comment]
@liangxia no, just make a copy and lazy to delete, and the newlines are harmless.

File: image-streams/tc510523.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+    },
+
+    "spec": {
+
+       "dockerImageRepository": "aosqe/ruby-20-centos7",
+
+            "tags":[
+
+                       { "name": "3.3",
```
Comment (wsun1) [inline image-streams/tc510523.json]: for your ruby image,is there 3.3 tag?

File: image-streams/tc510526.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
+{
+
+    "kind": "ImageStream",
+
+    "apiVersion": "v1",
+
+    "metadata": {
+
+      "name": "python",
```
Comment (wsun1) [inline image-streams/tc510526.json]: Your image is ruby,but your name is python,or you could use more meaningful name?

File: image-streams/tc510526.json [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+      "creationTimestamp": null
+
+    },
+
+    "spec": {
+
+
+           "tags":[
+
+                      { "name": "3.3",
```
Comment (wsun1) [inline image-streams/tc510526.json]: 3.3 tag?

File: image-streams/tc510523.json (hunk 3 above)
Comment (wjiangjay) [inline image-streams/tc510523.json]: The name is just a identity string, any one will be ok

File: image-streams/tc510528.json [hunk 7]
```diff
@@ -0,0 +1,20 @@
+{
+    "kind": "ImageStream",
+    "apiVersion": "v1",
+    "metadata": {
+      "name": "python",
```
Comment (wsun1) [inline image-streams/tc510528.json]: Your image is ruby,but your name is python,or you could use more meaningful name?

File: image-streams/tc510528.json [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+      "creationTimestamp": null
+    },
+    "spec": {
+      "dockerImageRepository": "aosqe/hello-openshift",
+           "tags":[
+                      { "name": "3.3",
```
Comment (wsun1) [inline image-streams/tc510528.json]: 3.3 tag?

**Pull request merged**